src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/visits/VisitResponseTest.kt

```diff
@@ -152,14 +152,14 @@ internal class VisitResponseTest {
                     person = this,
                     phoneType = PhoneUsage("HOME", "Home"),
                     phoneNo = "0123456789",
-                  ),
+                  ).apply { createDatetime = LocalDateTime.now() },
```

Comment:
unit test now needs to manually set date since now a generated field

Code only needed since this is a unit test not an integration test